code-challenges.js

```diff
@@ -9,60 +9,155 @@
 // Describe your process:
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+//const fruit1 = "apple"
+//const fruit2 = "banana"
 // Expected outcome: "banana"
 
 // Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+//const fruit3 = "cherry"
+//const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
+            //  Creating this function, I will utilize the Boolean primitive data type. I spent the most time on this code with a failed results.
+// I created a function using the provided variables. I created a const |boilPoint| to run everything in console.log.
+// The output reads |Cannot access 'temp1' before initialization|. I attempted different combinations and checked my syntax and have not been 
```

Comment:
This is a really helpful error!  The variables that you are using when you are invoking the function on lines 44 - 46 are declared after they are being called on.  The variables need to be declared before we use them.